pkg/coderefs/coderefs_test.go

```diff
@@ -6,11 +6,14 @@ import (
 	"strings"
 	"testing"
 
+	ignore "github.com/sabhiram/go-gitignore"
 	"github.com/stretchr/testify/require"
 
 	"github.com/launchdarkly/ld-find-code-refs/internal/ld"
 )
 
+var ignoreAll, _ = ignore.CompileIgnoreLines("*")
```

Comment:
Maybe you need a `MustCompileIgnoreLines` helper.  ;)

Reply:
I think it's fine considering it's compiling a hardcoded variable and we only use this once and only in tests. If we do end up using this more, I'll implement this feedback